packages/@navikt/person-card/src/cardStyles.less

```diff
@@ -0,0 +1,86 @@
+.card {
+    display: flex;
+    align-items: center;
+    padding: 0.625rem 2rem 0.375rem;
+    border-bottom: 0.25rem solid transparent;
+
+    &--active {
+        border-bottom: 0.25rem solid #0067c5;
+    }
+
+    &__container {
+        padding: 0 1rem;
+        display: flex;
+        align-items: center;
+    }
+
+    &__name {
+        &.typo-normal {
+            font-weight: 400;
+            padding: 0 1.5px;
+            color: #0067c5;
+        }
+
+        &--active {
+            text-decoration: none;
+            &.typo-normal {
+                font-weight: 600;
+                color: black;
+                padding: 0;
+            }
+        }
+    }
+
+    &__gender-icon {
+        width: 1.5rem;
+        height: 1.5rem;
+        margin-right: 1rem;
+
+        &--unknown {
+            width: 1.625rem;
+            height: 1.625rem;
+        }
+    }
+
+    &__selector {
+        background: transparent;
+        border: none;
+        display: flex;
+        align-items: center;
+        cursor: pointer;
+        padding: 0;
+        text-decoration: underline;
+        color: #0067c5;
+
+        &[aria-current='true'] {
+            text-decoration: none;
+        }
+
+        &:hover {
+            text-decoration: none;
+        }
+    }
+
+    &__menu-container {
+        border: 0.0625rem solid #b7b1a9;
+        position: absolute;
+        background-color: white;
+        left: -0.625rem;
+        padding: 1rem;
+    }
+
+    .arrow {
+        position: absolute;
+        background-color: white;
+        transform: rotate(45deg);
+        width: 1rem;
+        height: 1rem;
+
+        &[data-placement*='bottom-start'] {
+            border-top: 0.0625rem solid #b7b1a9;
+            border-left: 0.0625rem solid #b7b1a9;
+            top: -0.5rem;
+            z-index: 1;
```

Comment:
Trenger vi `z-index`? Det kan være uheldig å sette i en designkomponent mtp at det kompliserer bruken av `z-index` hos de som bruker komponenten.

Reply:
hvis ikke vi legger det inn tror jeg alle som skal bruke komponenten må legge det inn selv. Ellers kuttes pila i to